pkg/server/server.go

```diff
@@ -45,11 +45,11 @@ func newServer(ctx context.Context, path string) error {
 	return eg.Wait()
 }
 
-func createTmpDir() error {
+func CreateTempDir() error {
```

Comment:
any reason for making this method public? It is not really used anywhere else other than the server package